lab-lindsay/server.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const http = require('http');
+const Router = require('./lib/router');
+const storage = require('./lib/storage');
+const Candy = require('./model/candy');
+const debug = require('debug')('http:server');
+const PORT = process.env.PORT || 3000;
+
+const router = new Router();
+const server = module.exports = http.createServer(router.route());
+
+require('./routes/pokemon-routes');
```

Comment:
why require in pokemon routes? you don't have any pokemon. if you're using someone else's code as a map to get ideas, that's one thing. be careful with this...